evap/results/templates/results_index.html

```diff
@@ -83,7 +83,8 @@ <h6 class="card-subtitle mb-1">{% trans 'Semester' %}</h6>
                     <tbody>
                     {% regroup evaluations by course as course_evaluations %}
                     {% for course, evaluations in course_evaluations|dictsort:"grouper.name" %}
-                        {% if course.evaluations.count > 1 %}
+                        {# we're actually using length here instead of count to avoid a query (`evaluations` is accessed below anyway #}
+                        {% if course.evaluations|length > 1 %}
```

Comment:
this is broken, it should be `course.evaluations.all`. i'll probably prefetch the count instead.